src/themes/components/card.ts

```diff
@@ -0,0 +1,30 @@
+import { createMultiStyleConfigHelpers } from '@chakra-ui/react';
+import { cardAnatomy } from '@chakra-ui/anatomy';
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(cardAnatomy.keys);
+
+const baseStyle = definePartsStyle({
+  container: {
+    backgroundColor: 'white',
+    boxShadow: 'xs',
+    borderRadius: 'sm',
+    overflow: 'hidden',
+  },
+  header: {
+    paddingX: '2xl',
+    marginY: '2xl',
+  },
+  body: {
+    paddingX: '2xl',
+    marginY: '2xl',
+  },
+  footer: {
+    borderTop: '1px solid',
```

Comment:
I think you can go only with `1px` from our tokens

```suggestion
    borderTop: '1px',
```